src/bundle/Resources/public/js/scripts/helpers/timezone.helper.js

```diff
@@ -1,29 +1,29 @@
 (function(global, doc, eZ, moment) {
     const userPreferedTimezone = eZ.adminUiConfig.timezone;
-    const userPreferedFullDateFormat = eZ.adminUiConfig.dateFormat.full;
```

Comment:
`eZ.adminUiConfig.dateFormat.full` is globally available. Removing this is a BC break.

Reply:
@dew326 Is right. We can't change the name in UI Config (PHP). Just align your code to `full` and `short` names.